tests/pipelines/main.nf.test

```diff
@@ -66,6 +69,139 @@ nextflow_pipeline {
         }
     }
 
+    test("Small-scale test of full pipeline hamming distances") {
+        tag "pipeline_hamming"
+
+        when {
+            params {
+                input = "$baseDir/tests/data/samplesheets/samplesheet-hamming.csv"
+                outdir = "results"
+
+                gm_thresholds = "2,1,0"
+
+                metadata_1_header = "myheader_1"
+                metadata_2_header = "myheader_2"
+                metadata_3_header = "myheader_3"
+                metadata_4_header = "myheader_4"
+                metadata_5_header = "myheader_5"
+                metadata_6_header = "myheader_6"
+                metadata_7_header = "myheader_7"
+                metadata_8_header = "myheader_8"
```

Comment:
I think you can leave this in if you want, but you could also have them as the defaults (and not explicitly test them for test compactness / readability). You'll have to update the ArborView HTML check later in this test though. Either way is fine.

Reply:
That's a good point. I've updated to check the default metadata headers instead in this test in 875113b41bd396fd15b1dbfac671830345ff6e32